presto-cli/src/main/java/com/facebook/presto/cli/Query.java

```diff
@@ -279,20 +283,34 @@ private void pageOutput(OutputFormat format, List<String> fieldNames)
                 });
             }
             handler.processRows(client);
+            //recordClientStop. We do this *before* we close the OutputHandler and Pager
```

Comment:
nit: 
// Record the CLI query end time *before* closing the handler and pager.
Also add an empty line above